policies/templates/gcp_vpc_sc_ensure_access_levels_v1.yaml

```diff
@@ -26,8 +26,18 @@ spec:
       validation:
         openAPIV3Schema:
           properties:
-            required_access_levels:
-              description: "Required access level names"
+            mode:
+              type: string
+              enum: [whitelist, blacklist, reqired]
```

Comment:
type: reqired/required, do you mean require here (used below in the code)

Reply:
Enum has been corrected